backend.py

```diff
@@ -318,6 +318,10 @@ def performInstallation(answers, ui_package, interactive):
                                                          default_host_config['dom0-mem'])
                 default_host_config['dom0-vcpus'] = xcp.dom0.default_vcpus(hardware.getHostTotalCPUs(),
                                                                            answers['host-config']['dom0-mem'])
+
+            # Set xen-pciback if necessary.
+            if 'xen-pciback.hide' in answers['host-config']:
```

Comment:
Why is it necessary to set `default_host_config` here?

`default_host_config` is used to populate `answers` below, but `'xen-pciback.hide'` is already in `answers.

Reply:
It looks like a mistake indeed.